test/CMakeLists.txt

```
@@ -100,113 +100,76 @@ else()
 endif()
 
 #-----------------------------------------------------------------------------#
-# Gtest additions
-celeritas_setup_tests(SERIAL PREFIX gtest/detail)
+# Testing additions
+celeritas_setup_tests(SERIAL PREFIX test/detail)
 
-celeritas_add_test(gtest/detail/test/Macros.test.cc)
+celeritas_add_test(detail/Macros.test.cc)
 
 #-----------------------------------------------------------------------------#
-# Base
-
-celeritas_setup_tests(SERIAL PREFIX base)
-
-celeritas_add_test(base/Algorithms.test.cc)
-celeritas_add_test(base/Array.test.cc)
-celeritas_add_test(base/ArrayUtils.test.cc)
-celeritas_add_test(base/Constants.test.cc ${_optional_geant4_link})
-celeritas_add_test(base/Copier.test.cc GPU)
-celeritas_add_test(base/DeviceAllocation.test.cc GPU)
-celeritas_add_test(base/DeviceVector.test.cc GPU)
-celeritas_add_test(base/Join.test.cc)
-celeritas_add_test(base/OpaqueId.test.cc)
-celeritas_add_test(base/Quantity.test.cc)
-celeritas_add_test(base/Repr.test.cc)
-celeritas_add_test(base/ScopedStreamRedirect.test.cc)
-celeritas_add_test(base/SoftEqual.test.cc)
-celeritas_add_test(base/Span.test.cc)
-celeritas_add_test(base/TypeDemangler.test.cc)
-celeritas_add_test(base/VectorUtils.test.cc)
-
-celeritas_add_test(base/Stopwatch.test.cc ADDED_TESTS _test_name)
-set_tests_properties(${_test_name} PROPERTIES LABELS "nomemcheck")
-
-celeritas_device_test(base/Collection)
-celeritas_device_test(base/NumericLimits)
-celeritas_device_test(base/Range)
-celeritas_device_test(base/StackAllocator)
-
-#-----------------------------------------------------------------------------#
-# Comm
-
-celeritas_setup_tests(PREFIX comm)
-
-celeritas_add_test(comm/Communicator.test.cc)
-celeritas_add_test(comm/Environment.test.cc
+# Corecel
+celeritas_setup_tests(SERIAL PREFIX corecel)
+
+# Cont
+celeritas_add_test(corecel/cont/Array.test.cc)
+celeritas_add_test(corecel/cont/Span.test.cc)
+celeritas_device_test(corecel/cont/Range)
+
+# Data
+celeritas_device_test(corecel/data/Collection)
+celeritas_add_test(corecel/data/Copier.test.cc GPU)
+celeritas_add_test(corecel/data/DeviceAllocation.test.cc GPU)
+celeritas_add_test(corecel/data/DeviceVector.test.cc GPU)
+celeritas_add_test(corecel/data/OpaqueId.test.cc)
```

```suggestion
celeritas_add_test(corecel/OpaqueId.test.cc)
```
(and moving the file) since OpaqueId has moved (if I recall correctly)

---

Good catch, thanks!